src/pyrtma/compilers/python.py

```diff
@@ -319,12 +286,20 @@ def generate(self, out_filepath: pathlib.Path):
 
             f.write("# Struct Definitions\n")
             for obj in self.parser.struct_defs.values():
-                f.write(self.generate_struct(obj))
+                if obj.name in core_defs:
```

Comment:
This needs an exception for if we are actually compiling the core defs from core_defs.yaml. But we also discussed how we should probably just revert to copying the core defs to user defs and replace with core defs classes in runtime internally within the client.